core/trino-server/src/main/provisio/trino.xml

```diff
@@ -33,101 +33,101 @@
     </artifactSet>
 
     <!-- Plugins -->
-    <artifactSet to="plugin/bigquery">
+    <!-- <artifactSet to="plugin/bigquery">
```

Comment:
Thinking about future merges, would it be easier to keep these "begin / end comment" markers on separate lines, maybe?